internal/repositories/validate.go

```diff
@@ -0,0 +1,55 @@
+package repositories
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/DIMO-Network/telemetry-api/internal/graph/model"
+	"github.com/DIMO-Network/telemetry-api/internal/service/ch"
+)
+
+var (
+	twoWeeks       = 14 * 24 * time.Hour
+	errInvalidArgs = errors.New("invalid arguments")
+)
+
+func validateAggSigArgs(args *model.AggregatedSignalArgs) error {
+	if args.FromTS.IsZero() {
+		return fmt.Errorf("%w from timestamp is zero", errInvalidArgs)
```

Comment:
This wrap feels a bit odd to me, especially since we're not exporting the wrapped thing. Maybe a new struct?

```
type ValidationError string

func (e ValidationError) Error() string { return fmt.Sprintf("invalid argument: %s", e) }
```
(I don't actually know how well this works with `errors.As`).